src/index.css

```diff
@@ -0,0 +1,174 @@
+body {
+  background-color: rgb(202, 44, 155);
+}
+
+.App {
+  padding: 0;
+  margin: 0;
+  background-color: rgb(202, 44, 155);
+}
+
+.headline {
+  font-size: 2em;
+  color: white;
+  text-align: center;
+  margin-top: 0;
+  margin-bottom: 10px;
+}
+
+.data {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(33.33%, 1fr));
+  grid-auto-rows: 8%;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  border-top: 1px solid #cff;
+}
+
+.itemSpan {
+  position: absolute;
+  top: 50%;
+  font-size: x-small;
+  color: rgb(171, 198, 219);
+  gap: 10px;
+}
+
+.colDiv:nth-child(odd) {
+  background-color: rgb(202, 44, 155);
+  width: 100%;
+  height: 100%;
+}
+
+.colDiv:nth-child(even) {
+  background-color: rgb(172, 40, 117);
+  width: 100%;
+  height: 100%;
+}
+
+.colDiv button {
+  border: none;
+  padding: 10px;
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+  text-align: right;
+  font-size: 20px;
+  display: flex;
+  color: #fff;
+  justify-content: end;
+  background: transparent;
+}
+
+.navBar ul {
```

Comment:
- [ ] I can see that you used nearly the same design for both home and details pages, So Please follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) for the layout of these two pages including the composition and space between elements.

Your design:

|  |  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/177792382-81970a79-a2ba-4e78-987d-a4aaec3805a2.png)  | ![image](https://user-images.githubusercontent.com/68796232/177792418-4ff05b43-1b71-461c-9576-c2872e72bf97.png)  |

Given design:

|  |  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/177792727-af60ef8b-e197-4126-a0df-fc9d0e26b703.png)  | ![image](https://user-images.githubusercontent.com/68796232/177792811-c1784a5e-1f1d-48aa-81ed-7b57a388b53d.png)  |